src/MADE.Media.Image/MADE.Media.Image.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="MSBuild.Sdk.Extras">
+
+  <PropertyGroup>
+    <TargetFramework>uap10.0.17763</TargetFramework>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <Product>MADE.NET Images</Product>
+    <Description>
+      This package includes image helpers such as:
+      - StorageFileThumbnailConverter for providing the ability to show a thumbnail for a storage file.
+    </Description>
+    <PackageTags>MADE UI Views Controls UWP FilePicker Input File</PackageTags>
```

Comment:
These package tags look like they were copied over from the `FilePicker` lib. Worth updating these.

Reply:
Updated the tags to reflect the library